horace_core/sqw/PixelData/@PixelData/move_to_page.m

```diff
@@ -25,9 +25,12 @@
 
 % -----------------------------------------------------------------------------
 function [page_number,total_num_pages, nosave] = parse_args(obj, varargin)
-parser = inputParser();
-parser.addRequired('page_number', @is_scalar_positive_int);
-parser.addParameter('nosave', false, @islognumscalar);
+persistent parser
```

Comment:
Again, is this at any point performance critical?

Reply:
this is certainly performance-critical function which should benefit from this modification